### DISCUSSION TITLE
Store the latest Build number from TestFlight in a variable in bash script.

### DISCUSSION THREAD

**Discussion by Ramanadeep** [fastlane/fastlane#17064] [Q&A]
Hello, 

is there a way I can get the latest build version number from testflight and store it in a variable in script? 
I have seen trying to use `latestTestflightBuildNumber` in fastfile using swift and also tried using 
`bundle exec fastlane run latest_testflight_build_number username:"****@***" app_identifier:"****" `
on terminal but the variable stores the whole response like in the attached screenshot. 
Can I just get the integer value out in anyway?

this is what I get in **BUILDNUMBER** variable

![Screen Shot 2020-08-18 at 12 53 45 pm](https://user-images.githubusercontent.com/13730991/90465392-a0d36d80-e152-11ea-8e02-dc07b73ede7f.png)

I just need the value **Result**

Thanks

**Comment by tuanngocptn** [comment]
Same question, thanks

**Comment by tuanngocptn** [comment]
I just wrote the bash script that will get the latest build number 
```bash
DATA_LOG=$(fastlane ios latest_testflight_build_number)
DATA_LOG=$(echo $DATA_LOG | grep --only-matching --extended-regexp '(is build: )([0-9]*)')
if [[ $DATA_LOG =~ (is build: )([0-9]*) ]]; then
  echo ${BASH_REMATCH[2]} #the lastest build number
else
  echo -1 #can not find the build number
fi
```
Hope that can help you

**Reply by LizCira** [reply]
Awesome, this saved me, I'm sure, a tremendous amount of work, thank you!! I did have a weird issue running the if loop, but I found to get the number alone (so I can increment it as part of a Bamboo build plan to upload builds to Testflight) I added another line (maybe the loop handles this, but posting in case someone else has my issue)

```
DATA_LOG =$(echo $DATA_LOG | tr -dc '0-9')
BUILD_NUMBER=$(( DATA_LOG +1))
```
